clang/test/Sema/c2x-fallthrough2.c

```diff
@@ -0,0 +1,18 @@
+// RUN: %clang_cc1 -fsyntax-only -std=c2x -verify %s
```

Comment:
```suggestion
// RUN: %clang_cc1 -fsyntax-only -std=c23 -verify %s
```
Maybe rename the file to be `c23` instead of `c2x` as well?